src/status_im/contexts/wallet/create_account/view.cljs

```diff
@@ -127,6 +127,7 @@
         {:size                :size-48
          :track-text          (i18n/label :t/slide-to-create-account)
          :customization-color @account-color
+         :on-close            #(rf/dispatch [:standard-auth/reset-login-password])
```

Comment:
This should be handled by `authorize`, as I don't see scenarios where we wouldn't want to reset the password after the bottom-sheet is closed. @J-Son89 ?

... but I'd avoid adding it in this PR as `authorize` is fully refactored already [over here](https://github.com/status-im/status-mobile/pull/18899/files#) and the conflicts might be hard to fix.

Reply:
@mmilad75 FYI added the password/error reset inside the authorization logic in this commit [41ca992](https://github.com/status-im/status-mobile/pull/18756/commits/41ca9921fc363fa775526b0037247abece31c247)

Reply:
@clauxx Thanks man. I used your code and removed the on-close on the slide-button component

Reply:
@clauxx The error also needed to be cleared, it was missing in your code, so I added it